testcontainers/tests/container.rs

```diff
@@ -0,0 +1,18 @@
+use testcontainers::{clients::Cli, images::hello_world::HelloWorld};
+
+#[test]
+fn container_should_be_send() {
+    let docker = Cli::default();
+    let node = docker.run(HelloWorld {});
+    need_send(node);
+}
+
+#[test]
+fn container_should_be_sync() {
+    let docker = Cli::default();
+    let node = docker.run(HelloWorld {});
+    need_sync(node);
+}
+
+fn need_send<T: Send>(_t: T) {}
+fn need_sync<T: Sync>(_t: T) {}
```

Comment:
We can combine this into one and make the test cheaper by just passing it as a type parameter using turbo fish:

```rust
assert_send_and_sync::<Container<...>>();
```

Reply:
Good point, done!